sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/ServiceAPITest.groovy

```diff
@@ -278,6 +278,11 @@ class ServiceAPITest extends APISpec {
 
         then:
         results.size() == 1
+        if (primaryBlobServiceClient.getServiceVersion() >= BlobServiceVersion.V2020_04_08) {
```

Comment:
is there a reason why we're specifically checking against the service version here? We generally always run tests against the latest version, so this may not be necessary